CONTRIBUTING.md

```diff
@@ -101,7 +101,10 @@ To contribute to MassaStation, you'll need to set up your development environmen
        ```
 
 3. Install Task
-Follow the installation instructions here: [task-install](https://taskfile.dev/installation/)
+Follow the installation instructions here: [task-install](https://taskfile.dev/installation/).
+
+On Windows, we recommend to run a Powershell as an administrator and install task with `go install github.com/go-task/task/v3/cmd/task@latest`.
+Then, use task in the Powershell as an administrator.
```

Comment:
I don't understand why you want people to run `go install` commands as administrator. (same for task tho)
Go is supposed to be installed for a specific user (or at least, store binaries in a directory of the user, by default `$HOME/go`)

Reply:
I just checked on my W11 computer, I don't need to run my terminal as administrator